src/theme/createSaleorTheme/createSaleorTheme.tsx

```diff
@@ -311,6 +312,7 @@ export const createTheme = (colors: SaleorThemeColors): SaleorTheme =>
       },
       caption: {
         fontSize: "1.2rem",
+        letterSpacing: 0,
```

Comment:
initial is `normal`, which in most cases renders the same as `0`, is there a reason behind this?

Reply:
`letter-spacing` is set globally to `0.02rem` so it resets it in this particular typography variant